cluster/gce/gci/configure-helper.sh

```diff
@@ -279,6 +279,38 @@ contexts:
   name: webhook
 EOF
   fi
+
+if [[ -n "${GCP_IMAGE_VERIFICATION_URL:-}" ]]; then
+    # This is the config file for the image review webhook.
+    cat <<EOF >/etc/gcp_image_review.config
```

Comment:
I was just placing them next to the /etc/gcp_authz.config and /etc/gcp_authn.config